client/src/pages/Eesha.jsx

```diff
@@ -1,14 +1,181 @@
+          <p className="eesha-sign-chart-h2">i have strong ideas about meaning and what's right or wrong.</p>
+          <p className="eesha-sign-chart-h2">i live to be free, and to learn.</p>
+        </div>
+
+        <div className="grid-two">
+          <img src={asc} alt="asc-sign" />
+          <p className="eesha-sign-chart-h1">ascendent || virgo || 11°45'54"</p>
+          <p className="eesha-sign-chart-h2">i am precise, diligent, peace-seeking, and organized.</p>
+          <p className="eesha-sign-chart-h2">i may seem a bit too attached to work, details, and the pursuit of perfection.</p>
+        </div>
+
+        <div className="grid-three">
+          <img src={moon} alt="moon-sign" />
+          <p className="eesha-sign-chart-h1">moon || taurus || 13°6'26"</p>
+          <p className="eesha-sign-chart-h2">i am deeply loyal to the people i care about.</p>
+          <p className="eesha-sign-chart-h2">i often feel powerless to fix messy situations.</p>
+        </div>
+
+        <div className="grid-four">
+          <img src={mer} alt="mer-sign" />
+          <p className="eesha-sign-chart-h1">mercury || capricorn || 10°59'53"</p>
+          <p className="eesha-sign-chart-h2">i tend to be a poor communicator (oops).</p>
+          <p className="eesha-sign-chart-h2">i am curious about and inclined to analyze my love life, happiness, and creative projects.</p>
+        </div>
+
+        <div className="grid-five">
+          <img src={ven} alt="ven-sign" />
+          <p className="eesha-sign-chart-h1">venus || scorpio || 10°20'31"</p>
+          <p className="eesha-sign-chart-h2">my romantic side is mysterious, passionate, and seductive..</p>
+          <p className="eesha-sign-chart-h2">aaaand that's enough of that.</p>
+        </div>
+
+        <div className="grid-six">
+          <img src={mar} alt="mar-sign" />
+          <p className="eesha-sign-chart-h1">mars || scorpio || 9°22'46"</p>
+          <p className="eesha-sign-chart-h2">i assert myself in a way that is serious and incisive.</p>
+          <p className="eesha-sign-chart-h2">once i decide i want to do something, i don't hold back.</p>
+        </div>
+
+        <div className="grid-seven">
+          <img src={jup} alt="jup-sign" />
+          <p className="eesha-sign-chart-h1">jupiter || leo || 17°52'53"</p>
+          <p className="eesha-sign-chart-h2">i grow and find understanding through magnanimity, inspiring confidence, thinking big, and feeling good about myself.</p>
+        </div>
+
+        <div className="grid-eight">
+          <img src={sat} alt="sat-sign" />
+          <p className="eesha-sign-chart-h1">saturn || gemini || 25°43'34"</p>
+          <p className="eesha-sign-chart-h2">i struggle with a lack of focus, restless energy, and getting bored easily.</p>
+          <p className="eesha-sign-chart-h2">no i don't.</p>
+        </div>
+
+        <div className="grid-nine">
```

Comment:
you shouldn't need a different className for each of these sign chart cards you got. they should all actually share the same className ideally.